src/nng_interface/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ add_library(nng_interface
 find_package(nng CONFIG REQUIRED)
 
 # Find flatbuffers library
-find_package(Flatbuffers CONFIG REQUIRED)
+find_package(FlatBuffers CONFIG REQUIRED)
```

Comment:
This is to use the FlatBuffers 2.0.8 which has changes to the output file name since 2.0.7 
https://github.com/google/flatbuffers/pull/7378/files